pkg/plugin/actions/actions.go

```diff
@@ -0,0 +1,64 @@
+package actions
+
+import (
+	"context"
+	"github.com/aquasecurity/starboard/pkg/generated/clientset/versioned"
+	"github.com/aquasecurity/starboard/pkg/kube"
+	"github.com/aquasecurity/starboard/pkg/kubehunter"
+	"github.com/aquasecurity/starboard/pkg/starboard"
+	"github.com/vmware-tanzu/octant/pkg/action"
+	"github.com/vmware-tanzu/octant/pkg/plugin/service"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/kubernetes"
+	"time"
+)
+
+const (
+	StarboardKubeHunterScan = "starboard.octant.dev/scanKubeHunter"
+
+	kubeHunterReportName = "cluster"
+)
+
+func ActionHandler(request *service.ActionRequest) error {
+	actionName, err := request.Payload.String("action")
+	if err != nil {
+		return err
+	}
+
+	switch actionName {
+	case StarboardKubeHunterScan:
+		alert := action.CreateAlert(action.AlertTypeInfo, "Creating kube-hunter report...", time.Second * 15)
+		request.DashboardClient.SendAlert(request.Context(), request.ClientState.ClientID(), alert)
+		return startKubeHunterScan(request.Context())
+	default:
+		// no-op
+	}
+	return nil
+}
+
+func startKubeHunterScan(ctx context.Context) error {
+	configFlags := genericclioptions.NewConfigFlags(true)
+	restconfig, err := configFlags.ToRESTConfig()
```

Comment:
I guess this in general may be a different client config than the one used by octant process. Is that correct?

Reply:
Yes, this is a good point and leaves more to be desired in the plugin interface.

A  workaround is to see if `$KUBECONFIG` is set and see if it matches the one from config flags since we can assume Octant looks at the same spot by default via `clientcmd.NewDefaultClientConfigLoadingRules().GetDefaultFilename()`.

Reply:
Thanks for clarification. I think it's a good start and it should work in most cases. We can also leave a TODO comment explaining when this approach may not work.